identity-apps-core/apps/authentication-portal/src/main/webapp/login.jsp

```diff
@@ -170,6 +170,8 @@
     boolean hasLocalLoginOptions = false;
     boolean isBackChannelBasicAuth = false;
     List<String> localAuthenticatorNames = new ArrayList<String>();
+    List<String> registeredLocalAuthenticators = Arrays.asList(BACKUP_CODE_AUTHENTICATOR,TOTP_AUTHENTICATOR,EMAIL_OTP_AUTHENTICATOR,MAGIC_LINK_AUTHENTICATOR,SMS_OTP_AUTHENTICATOR,OPEN_ID_AUTHENTICATOR,IDENTIFIER_EXECUTOR,JWT_BASIC_AUTHENTICATOR,BASIC_AUTHENTICATOR,IWA_AUTHENTICATOR,X509_CERTIFICATE_AUTHENTICATOR,FIDO_AUTHENTICATOR);
```

Comment:
Shall we span the authenticator list across multiple lines for better readability?

```suggestion
    List<String> registeredLocalAuthenticators = Arrays.asList(   
        BACKUP_CODE_AUTHENTICATOR, TOTP_AUTHENTICATOR, EMAIL_OTP_AUTHENTICATOR,
        MAGIC_LINK_AUTHENTICATOR,SMS_OTP_AUTHENTICATOR,OPEN_ID_AUTHENTICATOR,
        IDENTIFIER_EXECUTOR,JWT_BASIC_AUTHENTICATOR,BASIC_AUTHENTICATOR,
        IWA_AUTHENTICATOR,X509_CERTIFICATE_AUTHENTICATOR,FIDO_AUTHENTICATOR
   );
```